packages/lib/RotatingLogs.ts

```diff
@@ -0,0 +1,73 @@
+const moment = require('moment');
+const { FsDriverDummy } = require('./fs-driver-dummy.js');
+
+export default class RotatingLogs {
+
+	public static fsDriver_: any = null;
+
+	private logFileDir: string = '';
+	private logFileName: string = 'log.txt';
+	private logFileExpirationTimeInDays: number = 30;
+	private logFileMaximumSizeInBytes: number = 1024 * 1024 * 10;
+
```

Comment:
As a general rule, please rely on TypeScript implicit types. So here you don't need `:string`, `:number`, etc. It's implied from the assignment.